### ISSUE TITLE
@code_native debuginfo=:none shows comments

### ISSUE THREAD

**Issue by cossio** [JuliaLang/julia#31867]
I am getting a lot of comments from `@code_native debuginfo=:none`. Is that normal or a bug?

```
julia> @code_native 2+2
    .text
; ┌ @ int.jl:53 within `+'
    leaq    (%rdi,%rsi), %rax
    retq
    nopw    %cs:(%rax,%rax)
; └

julia> @code_native debuginfo=:none 2+2
    .text
; ┌ @ int.jl:53 within `+'
    leaq    (%rdi,%rsi), %rax
    retq
    nopw    %cs:(%rax,%rax)
; └
```

Shouldn't the second version eliminate all comments?

**Comment by vchuravy** [comment]
Which version of Julia are you on? This got fixed #30223

**Comment by cossio** [comment]
@vchuravy I am on Julia 1.1.0. I guess this will be fixed in Julia 1.1.1.
